lib/go-atscfg/regexrevalidatedotconfig.go

```diff
@@ -70,19 +70,19 @@ func MakeRegexRevalidateDotConfig(
 	}
 	warnings := []string{}
 
-	if server.CDNName == nil {
+	if &server.CDN == nil || server.CDN == "" {
 		return Cfg{}, makeErr(warnings, "server CDNName missing")
 	}
 
 	params := paramsToMultiMap(filterParams(globalParams, RegexRevalidateFileName, "", "", ""))
 
 	dsNames := map[string]struct{}{}
 	for _, ds := range deliveryServices {
-		if ds.XMLID == nil {
+		if &ds.XMLID == nil {
```

Comment:
This condition should be `ds.XMLID == ""`

Reply:
done